ivy/array/general.py

```diff
@@ -36,9 +36,9 @@ def is_native_array(
 
     def is_ivy_array(self: ivy.Array, /, *, exclusive: Optional[bool] = False) -> bool:
         """
-        ivy.Array instance method variant of ivy.is_ivy_array. This method simply 
-        wraps the function, and so the docstring for ivy.is_ivy_array also 
-        applies to this method with minimal changes.
+        ivy.Array instance method variant of ivy.is_ivy_array. This method simply
```

Comment:
Please revert this update as it's not related to the function you're reformatting.